Lib/test/test_embed.py

```diff
@@ -601,11 +603,6 @@ def get_expected_config(self, expected_preconfig, expected,
             if value is self.GET_DEFAULT_CONFIG:
                 expected_preconfig[key] = pre_config[key]
 
-        path_config = configs['path_config']
-        for key, value in expected_pathconfig.items():
-            if value is self.GET_DEFAULT_CONFIG:
-                expected_pathconfig[key] = path_config[key]
```

Comment:
We can get rid of the `expected_pathconfig` parameter completely, right?

Reply:
It's still doing the checks, just against a subset of the config instead of all of it (so you don't need to set up correct defaults for the rest if you're only testing the paths).

Reply:
I take that back - as you point out below, we're not ever checking it. I'll go through and clean it all up